profiles/community/github_mock/flaskapp.py

```diff
@@ -48,22 +52,26 @@
     'gh01': {
         'id': 1000,
         'login': 'gh01',
-        'password': 'redhat'
+        'password': 'redhat',
+        'email': 'gh01@gmail.com',
     },
     'gh02': {
         'id': 1001,
         'login': 'gh02',
-        'password': 'redhat'
+        'password': 'redhat',
+        'email': 'gh02@gmail.com',
     },
     'jctannerTEST': {
         'id': 1003,
         'login': 'jctannerTEST',
-        'password': 'redhat'
+        'password': 'redhat',
+        'email': 'jctannerTEST@gmail.com',
     },
     'geerlingguy': {
         'id': 1004,
         'login': 'geerlingguy',
-        'password': 'redhat'
+        'password': 'redhat',
+        'email': 'geerlingguy@gmail.com',
```

Comment:
Is it okay to use a real e-mail address here?

Reply:
I made it up ... I guess it could possibly be his real email?

Reply:
@jctanner it is his real e-mail